src/types/types.d.ts

```diff
@@ -398,3 +398,15 @@ export interface RecommendedCallUsed {
   params: CallParameterDefinition[];
   decodeText: string;
 }
+
+export type ProposalsExtended = Proposal &

```

Comment:
When I worked on it I used this type:
```
  interface ProposalStatus {
    boostTime: BigNumber;
    finishTime: BigNumber;
    status: string;
    pendingAction: number;
  }
  ```
 I think if you make `ProposalExtended = Proposal & ProposalStatus;` it will work.